File: src/main/java/com/team3/gudit/payment/config/TossPaymentProperties.java
package com.team3.gudit.payment.config;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties(prefix = "toss.payments")
public class TossPaymentProperties {

    private String clientKey;
    private String secretKey;
}

File: src/main/kotlin/com/team3/gudit/payment/config/TossPaymentProperties.kt
package com.team3.gudit.payment.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "toss.payments")
class TossPaymentProperties {
    var clientKey: String? = null
    var secretKey: String? = null
}
